lib/dpkg_database.rb

```diff
@@ -45,6 +45,21 @@ def handle_verify_fail(_path)
     # This method is a no-op for DpkgDatabase.
   end
 
+  def determine_changes(line)
```

Comment:
is it possible to just have a parse_changes_line method for dpkg that does this logic and then calls super?